pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/HelixBrokerStarter.java

```diff
@@ -319,6 +322,8 @@ public void start()
             new BrokerUserDefinedMessageHandlerFactory(_routingManager, queryQuotaManager));
     _participantHelixManager.connect();
     addInstanceTagIfNeeded();
+    updateHelixHost(_brokerConf, _participantHelixManager, _clusterName, _brokerId);
```

Comment:
Shall we make it consistent with server by merging `addInstanceTagIfNeeded()` and `updateHelixHost()` to `updateInstanceConfigIfNeeded()`? The logic should be very similar. Same for controller and minion

Reply:
Yes, a good point; I refactored the code so now the 3 host types are using the same function now.